views/index-copy.ejs

```diff
@@ -0,0 +1,125 @@
+            <a class="navbar-brand" href="#carousel-container">WALL OF WISDOM</a>
+
+
+            <!-- Collapse button -->
+            <button class="navbar-toggler pull-right hamburger" type="button" data-toggle="collapse" data-target="#navbarSupportedContent20" aria-controls="navbarSupportedContent20"
+                aria-expanded="false" aria-label="Toggle navigation">
+                <div class="animated-icon1">
+                    <span></span>
+                    <span></span>
+                    <span></span>
+                </div>
+            </button>
+
+            <!-- Collapsible content -->
+            <div class="collapse navbar-collapse" id="navbarSupportedContent20">
+
+                <!-- Links -->
+                <ul class="nav navbar-nav">
+                    <li class="nav-item">
+                        <a class="nav-link" href="#carousel-slides">Quotes<span class="sr-only">(current)</span></a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#form">Add</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#about">About</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#footer">Contact</a>
+                    </li>
+                </ul>
+                <!-- Links -->
+            </div>
+            <!-- Collapsible content -->
+        </div>
+    </nav>
+    <!-- End of Navbar -->
+
+
+    <video autoplay muted loop id="wall-video">
+      <source src="images/wall.mp4" type="video/mp4">
+      Your browser does not support HTML5 video.
+    </video>
+
+
+    <!-- Start of Carousel of Quotes -->
+    <div id="carousel-container" class="container">
+        <div class="row">
+            <div class="col-md-12">
+                <div id="carousel-example-generic" class="carousel slide" data-ride="carousel">
+                    <!-- Indicators -->
+                    <ol class="carousel-indicators">
+                        <li data-target="#carousel-example-generic" data-slide-to="0" class="active"></li>
+                        <% for (let j = 1; j < users.length && j < 5; j++) { %>
+                            <li data-target="#carousel-example-generic" data-slide-to="<% j %>"></li>
+                        <% } %>
+                    </ol>
+
+                    <!-- Wrapper for slides -->
+                    <div id="carousel-slides" class="carousel-inner" role="listbox">
+                        <div class="item active quotes center-block text-justified">
+                            <h1><%= users[users.length-1].quote %></h1>
+                            <p><%= users[users.length-1].name %></p>
+                            <br>
+                            <div>
+                                <img class="heartEmpty pull-right" src="images/heartEmpty.png" alt="like?"/>
```

Comment:
For the `alt` text of images it’s a good idea to describe the image or its purpose. For example you can have `alt="Next"` and `alt="Back"`. These are good for accessibility, for example visually impaired people who are using screen readers that will read out the content of the `alt` tags.